clar2wasm/src/lib.rs

```diff
@@ -10,13 +10,11 @@ use clarity::{
 use walrus::Module;
 use wasm_generator::{GeneratorError, WasmGenerator};
 
+pub mod datastore;
```

Comment:
This should also be only used for tests.

Reply:
fixed